kolibri/content/models.py

```diff
@@ -170,11 +177,14 @@ def save(self, *args, **kwargs):
             md5 = hashlib.md5()
             for chunk in self.content_copy.chunks():
                 md5.update(chunk)
-
             self.checksum = md5.hexdigest()
             self.available = True
             self.file_size = self.content_copy.size
-            self.extension = os.path.splitext(self.content_copy.name)[1]
+            # if feed a format that doesn't exist yet will break the app
+            # not sure how we want to handle this yet
```

Comment:
We should probably follow the conventions of Django REST Framework, and do something like:

![image](https://cloud.githubusercontent.com/assets/618416/15190227/9a7e3e50-1763-11e6-87e6-9c78432e7e01.png)

Except in this case, the status code would be "400 Bad request", and the "detail" could be something like "'EXE' is not a supported file format."